hub/policy/src/main/java/uk/gov/ida/hub/policy/configuration/RedisConfiguration.java

```diff
@@ -19,11 +20,19 @@
     @JsonProperty
     private URI uri;
 
+    @Valid
+    @JsonProperty
+    private Duration timeout = Duration.of(20L, SECONDS);
+
     public Long getRecordTTL() {
         return recordTTL.getSeconds();
     }
 
     public RedisURI getUri() {
-        return RedisURI.Builder.redis(uri.getHost(), uri.getPort()).withTimeout(Duration.ofSeconds(20L)).build();
```

Comment:
`setDefaultTimeout(redisURI.getTimeout());` also gets called during the ` RedisClient redisClient = RedisClient.create()` but I guess leaving out the one for SAML Engine would have caused problems. I'll approve this